doc/man3/EVP_PKEY_is_a.pod

```diff
@@ -48,6 +57,10 @@ supports signing, otherwise 0.
 
 EVP_PKEY_get0_type_name() returns the name that is found or NULL on error.
 
+EVP_PKEY_get0_description() returns the name that is found or NULL if not.
+
+EVP_PKEY_get0_provider() returns the provider that is found or NULL if not.
```

Comment:
s/is/if/

Reply:
Actually, s/that is/if/, no?

Reply:
Yes.  "found" isn't an ideal word either but I can't think of anything better off hand.